tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -115,7 +111,9 @@ def check_thresh(check_vals):
     diag_elems = jax.numpy.zeros(ncv, dtype=init.dtype)
 
     norms = jax.ops.index_update(norms, jax.ops.index[0], 1.0)
-    norms_dtype = np.real(init.dtype).dtype
+
+    norms_dtype = jax.numpy.real(jax.numpy.empty((0, 0),
+                                                 dtype=init.dtype)).dtype
```

Comment:
revert this.

Reply:
the old code broke on my system

Reply:
Why? What was the error?

Doing it this way is very hacky. There must be a better way.

Reply:
it works now so I reverted it.